src/actions/MapActions.js

```diff
@@ -1,20 +1,29 @@
+import firebase from '@firebase/app';
+import '@firebase/firestore'
```

Comment:
These libraries are not used in this PR but will be necessary for GH-26 which is branched off this branch.